bom/application/pom.xml

```diff
@@ -14,6 +14,7 @@
     <packaging>pom</packaging>
 
     <properties>
+        <bitbucket.version>0.7.12</bitbucket.version>
```

Comment:
This one probably needs to be added to Dependabot? Or put together with the dependencies that need alignment if it needs to be aligned (together with comments indicating the need for alignment).

Reply:
@sberyozkin I added a direct dependency on jose4j to the module using its API directly. So it is both a direct dependency and a transitive dependency through `io.smallrye:smallrye-jwt-build`. Would you like to control it in the `quarkus-bom` or use whatever `smallrye-jwt-build` depends on? In the later case, perhaps we shouldn't add it to the BOM, unless you think the probability of conflicts will be high. `smallrye-jwt-build` is already managed, so if it comes only as a dependency of `smallrye-jwt-build` there won't be a problem if don't add it to the BOM.